schema/types/base/properties.pb.go

```diff
@@ -0,0 +1,160 @@
+// Code generated by protoc-gen-go. DO NOT EDIT.
+// versions:
+// 	protoc-gen-go v1.26.0
+// 	protoc        v3.17.3
+// source: proto/persistence/schema/types/base/v1beta1/properties.proto
+
+package base
+
+import (
+	//"github.com/persistenceOne/persistenceSDK/schema/types"
+	"github.com/cosmos/cosmos-sdk/codec/types"
+	_ "github.com/regen-network/cosmos-proto"
+	protoreflect "google.golang.org/protobuf/reflect/protoreflect"
+	protoimpl "google.golang.org/protobuf/runtime/protoimpl"
+	anypb "google.golang.org/protobuf/types/known/anypb"
+	reflect "reflect"
+	sync "sync"
+)
+
+const (
+	// Verify that this generated code is sufficiently up-to-date.
+	_ = protoimpl.EnforceVersion(20 - protoimpl.MinVersion)
+	// Verify that runtime/protoimpl is sufficiently up-to-date.
+	_ = protoimpl.EnforceVersion(protoimpl.MaxVersion - 20)
+)
+
+type Properties struct {
+	state         protoimpl.MessageState
```

Comment:
we can make a common proto struct with these three members and add to all proto structs

Reply:
Yes we can make one and use it in other proto files. This was while we were experimenting things.